### PR TITLE
Added properties to validate saml request and the certificate

### PR DESCRIPTION
**Pull request by GDRDABARERA**
Resolves wso2/product-is#5883 and wso2/product-is#5891

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 1]
```diff
@@ -642,6 +642,10 @@
         <SAMLECPEndpoint>{{saml.endpoints.ecp}}</SAMLECPEndpoint>
         <SAMLMetadataValidityPeriod>{{saml.metadata.validity_period}}</SAMLMetadataValidityPeriod>
         <SAMLMetadataSigningEnabled>{{saml.metadata.enable_signing}}</SAMLMetadataSigningEnabled>
+        <SAML2AuthenticationRequestValidityPeriodEnabled>{{saml.request_validity_period_enabled}}</SAML2AuthenticationRequestValidityPeriod>
```
Comment (hasinidilanka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Can we change the key to enable_request_validity_period

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 2]
```diff
@@ -642,6 +642,10 @@
...
+        <!-- Request validity period in minutes-->
+        <SAML2AuthenticationRequestValidityPeriod>{{saml.request_validity_period}}</SAML2AuthenticationRequestValidityPeriod>
+        <SAML2RequestCertificateExpiryValidationEnabled>{{saml.request_certificate_expiry_validation_enabled}}</SAML2RequestCertificateExpiryValidationEnabled>
```
Comment (hasinidilanka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Can we change the key to enable_request_certificate_expiry_validation

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 3]
```diff
@@ -642,6 +642,10 @@
         <SAMLECPEndpoint>{{saml.endpoints.ecp}}</SAMLECPEndpoint>
         <SAMLMetadataValidityPeriod>{{saml.metadata.validity_period}}</SAMLMetadataValidityPeriod>
         <SAMLMetadataSigningEnabled>{{saml.metadata.enable_signing}}</SAMLMetadataSigningEnabled>
+        <SAML2AuthenticationRequestValidityPeriodEnabled>{{saml.enable_request_validity_period}}</SAML2AuthenticationRequestValidityPeriod>
```
Comment (darshanasbg) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Close tag is wrong here... It causes a startup error.

Comment (darshanasbg) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Fixed with: https://github.com/wso2/carbon-identity-framework/pull/2303/files

**Pull request merged**